<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResetPasswordPage</class>
 <widget class="QWidget" name="ResetPasswordPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">  QWidget {
      background-color: rgb(231, 236, 239);;
  }</string>
  </property>
  <widget class="QLineEdit" name="passwordLineEdit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>410</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  QWidget {
      background-color: #ffffff;
  }</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirmPasswordLineEdit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>470</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  QWidget {
      background-color: #ffffff;
  }</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updatePasswordButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>540</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  QWidget {
      background-color: #00ffff;
  }</string>
   </property>
   <property name="text">
    <string>Update Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>70</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">  QLabel {
      color: #003380;
  }</string>
   </property>
   <property name="text">
    <string>Reset Password </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>390</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>New Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>450</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm New Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>150</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>We recommend for your password to be 
atleast 15 characters. Your password should 
also be unique and difficult to guess. </string>
   </property>
  </widget>
  <widget class="QPushButton" name="showPasswordButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>440</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: #6ea8fe;</string>
   </property>
   <property name="text">
    <string>Show</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
